proto/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "miden-node-proto"
-version = "0.1.0"
 authors = ["miden contributors"]
-readme = "README.md"
+edition = "2021"
+keywords = ["miden", "node", "program", "proto"]
 license = "MIT"
+name = "miden-node-proto"
+readme = "README.md"
 repository = "https://github.com/0xPolygonMiden/miden-node"
-keywords = ["miden", "node", "program", "proto"]
-edition = "2021"
 rust-version = "1.67"
```

Comment:
We usually don't sort the `package` section (just the dependencies). So, my preference would be to keep this part as it was to make it consistent with all other creates.